src/v2/Apps/Partner/Components/PartnerArticles/helpers/articlesCard.ts

```diff
@@ -0,0 +1,35 @@
+import lodashMap from "lodash/map"
+import { getENV } from "v2/Utils/getENV"
+
+export const toHumanSentence = (list): string => {
+  return list.join(", ").replace(/,\s([^,]+)$/, ", and $1")
+}
+
+export const getAuthors = (
+  channelID: string,
+  author: { name: string },
+  contributingAuthors: readonly { readonly name: string }[]
+): { authorName: string; editorialName: string | null } => {
+  const authorNameObj = {
+    authorName: null,
+    editorialName: "artsy editorial",
+  }
+
+  const isIDsMatched = getENV("ARTSY_EDITORIAL_CHANNEL") === channelID
+
+  if (!isIDsMatched) {
+    authorNameObj.editorialName = null
+  }
+
+  if (contributingAuthors.length) {
+    const authorName = lodashMap(contributingAuthors, "name")
```

Comment:
Forgive me, not trying to nitpick here but trying to understand — why use a library when we have a native map which is much easier to read at a glance?

```suggestion
    const authorName = contributingAuthors.map(author => author.name)
```

Reply:
@dzucconi  fair enough. My bad.